### PR TITLE
[staging-next] libpsl: 0.21.0 -> 0.21.1

### PR DESCRIPTION
**Pull request by trofi**

### CODE REVIEW

File: pkgs/development/libraries/libpsl/default.nix [hunk 1]
```diff
@@ -21,11 +21,11 @@ let
     && !stdenv.isAarch64;
 in stdenv.mkDerivation rec {
   pname = "libpsl";
-  version = "0.21.0";
+  version = "0.21.1";
 
   src = fetchurl {
-    url = "https://github.com/rockdaboot/${pname}/releases/download/${pname}-${version}/${pname}-${version}.tar.lz";
-    sha256 = "183hadbira0d2zvv8272lspy31dgm9x26z35c61s5axcd5wd9g9i";
+    url = "https://github.com/rockdaboot/${pname}/releases/download/${version}/${pname}-${version}.tar.lz";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libpsl/default.nix:27]:
```suggestion
    url = "https://github.com/rockdaboot/libpsl/releases/download/${version}/libpsl-${version}.tar.lz";
```

**Comment by r-burns** [comment]
This fixes libpsl on staging-next, would you retarget please?

**Comment by trofi** [comment]
Rebased against `staging-next`.

**Comment by jonringer** [comment]
looks like i came to the same conclusion: https://github.com/NixOS/nixpkgs/pull/146332

**Review by jonringer — APPROVED** [review]
LGTM

```
https://github.com/NixOS/nixpkgs/pull/146255

1 package built:
libpsl
```

**Pull request merged**